### DISCUSSION TITLE
Deep Space Network plugin

### DISCUSSION THREAD

**Discussion by tobiasbrown** [nasa/openmct#4616] [Show and tell]
Hi Open MCT,

I've built the [openmct-dsn](https://github.com/tobiasbrown/openmct-dsn) plugin to integrate telemetry from the [Deep Space Network](https://eyes.nasa.gov/dsn/).  This was completed as a response to #1656.  Thank you to everyone that provided feedback particularly @akhenry and @jvigliotta.

If you end up using this plugin, please share as I'd love to know how it's being used!

📡 〰️ 〰️ 〰️ 〰️ 〰️ 🛰️

**Comment by unlikelyzero** [comment]
Great work!

**Comment by jvigliotta** [comment]
That's awesome! Can't wait to try it out!

**Comment by iSagnik** [comment]
@tobiasbrown  Hi, I am developing a project with OpenMCT as well that involves a timeline and was having trouble adding some plugins/custom features. I would appreciate it if I could have a one on one discussion with you to ask some questions if you can make the time! This is project os for a student rocketry team called Liquid Rocketry Lab ([Liquid Rocketry Lab](liquidrocketry.com))
